.golangci.yml

```diff
@@ -14,7 +14,7 @@ linters-settings:
   errcheck:
     check-type-assertions: true
     check-blank: true
-
+    
```

Comment:
Let's remove the spaces here

Reply:
Removing the space doesn't cause the file to not track any changes. Odd.

Reply:
Fixed it!